content/en/docs/Badges/MCH2022/software-development/fpga/_index.md

```diff
@@ -15,6 +15,8 @@ python3 mch2022-tools/webusb_fpga.py mch2022-firmware-ice40/projects/Hello-World
 
 If the TL;DR wasn't wordy enough for you, try ["FPGA **Getting Started** for Badgers with Tiny Brains"](fpga-getting-started) or read on!
 
+If you look for a beginner friendly, graphical FPGA development suite: [https://github.com/badgeteam/mch2022-icestudio](https://github.com/badgeteam/mch2022-icestudio)
```

Comment:
I find this a bit confusing tbh... This says "download the WIP version from download.icestudio.io", which redirects to https://github.com/FPGAwars/Collection-Jedi. There you find a  "stable" and "development" download ...

And those downloads releases contain source zips/tgz.

I think after all the work that went into this, it should be featured more prominently... If someone provides me a link to where to download, I'd be happy to do some screenshots and rudimentary docs ...

Reply:
Thanks @a2800276 !  I had fixed the links.

I'm working in an more detailed "started guide" this a temporal readme until i finish it, don't worry.

The zip into the zip in the  linux binary is an improvement that i need to do in our CI flow with github actions, i'll try to fix asap.

Thanks again for the feedback!

Reply:
Thanks again! Please let me know if I can do anything to help.